tracer/src/Datadog.Trace.Tools.Runner/RunCiCommand.cs

```diff
@@ -198,7 +219,7 @@ public override async Task<int> ExecuteAsync(CommandContext context, RunCiSettin
                     }
 
                     // Add the Datadog coverage collector
-                    var baseDirectory = Path.GetDirectoryName(typeof(Coverage.Collector.CoverageCollector).Assembly.Location);
+                    var baseDirectory = AppContext.BaseDirectory;
```

Comment:
These aren't necessarily the same thing, right? I'm guessing you did this because of AOT issues?

Reply:
It's the analyzer that suggested the change (for AOT). It's not the current directory but the directory that assemblies are resolved from: https://learn.microsoft.com/en-us/dotnet/api/system.appcontext.basedirectory?view=net-7.0
In the case of our tool I believe this what we want (since all assemblies are in the same folder).